src/gei/Commands/MigrateOrgCommand.cs

```diff
@@ -92,7 +93,29 @@ public async Task Invoke(MigrateOrgCommandArgs args)
                 sourceToken,
                 targetToken);
 
-            _log.LogSuccess($"Org Migration has been initiated (ID: {migrationId}).");
+
+            if (!args.Wait)
+            {
+                _log.LogInformation($"A organization migration (ID: {migrationId}) was successfully queued.");
+                return;
+            }
+
+            var migrationState = await githubApi.GetOrganizationMigrationState(migrationId);
+
+            while (OrganizationMigrationStatus.IsPending(migrationState))
+            {
+                _log.LogInformation($"Migration in progress (ID: {migrationId}). State: {migrationState}. Waiting 10 seconds...");
+                await Task.Delay(10000);
+                migrationState = await githubApi.GetOrganizationMigrationState(migrationId);
+            }
+
+            if (OrganizationMigrationStatus.IsFailed(migrationState))
+            {
+                _log.LogError($"Migration Failed. Migration ID: {migrationId}");
+                throw new OctoshiftCliException($"Migration Failed. Migration ID: {migrationId}");
```

Comment:
Can we get the failureReason from octoshift and display this to the user here? That's what we do in migrate-repo

Reply:
I was not able to find the equivalent in the graphql classes. I will check with @juanmrad to see if it exists or will be implemented.